azblob/list.go

```diff
@@ -16,12 +16,22 @@ func (azp *Storer) Count(ctx context.Context, tagsFilter string, opts ...Option)
 
 	logger.Sugar.Debugf("Count")
 
-	r, err := azp.FilteredList(ctx, tagsFilter, opts...)
-	if err != nil {
-		return 0, err
-	}
-
-	return int64(len(r.Items)), nil
+	var count int64
```

Comment:
var marker string

is idiomatic and less confusing

Reply:
yes - correct code is in another PR